spec/models/bobswatches_item_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe BobswatchesItem, type: :model do
+  context 'validations' do
```

Comment:
here `context` change to `describe`, please. `describe` is more general then `context`